exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/StandardSerializers.java

```diff
@@ -16,125 +16,123 @@
 
 package com.exonum.binding.common.serialization;
 
-import static com.google.common.base.Preconditions.checkArgument;
-
+import com.exonum.binding.common.crypto.PrivateKey;
+import com.exonum.binding.common.crypto.PublicKey;
 import com.exonum.binding.common.hash.HashCode;
 import com.google.protobuf.CodedOutputStream;
 import com.google.protobuf.MessageLite;
-import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
-import java.nio.CharBuffer;
-import java.nio.charset.CharacterCodingException;
-import java.nio.charset.CharsetDecoder;
-import java.nio.charset.CodingErrorAction;
-import java.nio.charset.StandardCharsets;
 
 /**
  * A collection of pre-defined serializers.
  */
-// todo: test these guys with the code from exonum-serialization, when we migrate to JUnit5
-// (ECR-642)
 public final class StandardSerializers {
 
-  /** Returns a serializer of longs in little-endian byte order. */
-  public static Serializer<Long> longs() {
-    return LongSerializer.INSTANCE;
+  /**
+   * Returns a serializer that does nothing.
+   *
+   * @return no-op serializer
+   */
+  public static Serializer<byte[]> noOp() {
+    return NoOpSerializer.INSTANCE;
+  }
+
+  /**
+   * Returns a serializer of booleans in little-endian byte order.
+   *
+   * @return boolean serializer
+   */
+  public static Serializer<Boolean> bool() {
+    return BoolSerializer.INSTANCE;
+  }
+
+  /**
+   * Returns a serializer of fixed32 (integer) in little-endian byte order.
+   *
+   * @return fixed32 serializer
+   */
+  public static Serializer<Integer> fixed32() {
+    return Fixed32Serializer.INSTANCE;
+  }
+
+  /**
+   * Returns a serializer of fixed64 (long) in little-endian byte order.
+   *
+   * @return fixed64 serializer
+   */
+  public static Serializer<Long> fixed64() {
+    return Fixed64Serializer.INSTANCE;
   }
 
-  /** Returns a serializer of strings in UTF-8. */
+  /**
+   * Returns a serializer of floats in little-endian byte order.
+   *
+   * @return float serializer
+   */
+  public static Serializer<Float> floats() {
+    return FloatSerializer.INSTANCE;
+  }
+
+  /**
+   * Returns a serializer of doubles in little-endian byte order.
+   *
+   * @return double serializer
+   */
+  public static Serializer<Double> doubles() {
+    return DoubleSerializer.INSTANCE;
+  }
+
+  /**
+   * Returns a serializer of strings in UTF-8.
```

Comment:
Would you add please
`Deserializer will reject malformed input, but replace the characters not representable in UTF-16 with the default replacement character.`